charts/internal/machineclass/values.yaml

```diff
@@ -7,6 +7,7 @@ machineClasses:
   vnetName: my-vnet
   subnetName: my-subnet-in-my-vnet
   zone: 1
+  identityID: /subscriptions/subscription-id/resourceGroups/resource-group-name/providers/Microsoft.ManagedIdentity/userAssignedIdentities/identity-name
```

Comment:
```suggestion
# identityID: /subscriptions/subscription-id/resourceGroups/resource-group-name/providers/Microsoft.ManagedIdentity/userAssignedIdentities/identity-name
```

As the value is optional in the chart I suggest to make it optional here as well.